tchannel-core/src/main/java/com/uber/tchannel/api/DefaultRequestHandler.java

```diff
@@ -26,8 +26,10 @@
 
 public abstract class DefaultRequestHandler<T, U> implements RequestHandler<T, U> {
 
-    private TypeToken<T> requestType = new TypeToken<T>(getClass()) { };
-    private TypeToken<U> responseType = new TypeToken<U>(getClass()) { };
+    private TypeToken<T> requestType = new TypeToken<T>(getClass()) {
```

Comment:
As long as we conform the automated style checks I don't think we should be making these changes. Its just not scalable, so either push these stylechecks to the automated ones or lets not fix them.